### ISSUE TITLE
I can't scroll up!

### ISSUE THREAD

**Issue by 3m4r5** [hbang/NewTerm#80]
it automatically scrolls to the bottom preventing me from being able to read previous commands.

**Comment by kirb** [comment]
If anything in the terminal screen updates, it’ll scroll to bottom. Is it possible that’s what’s happening? If so, I do plan to solve that so it’ll only auto-scroll if you haven’t manually scrolled away from the bottom.

**Comment by 3m4r5** [comment]
@kirb no, it always scrolls to the bottom no matter what's happening.

**Comment by 3m4r5** [comment]
@kirb it only happens for the first swipe after you close the keyboard, but it's still annoying nonetheless.
